runtime/doc/starting.txt

```diff
@@ -506,12 +506,12 @@ accordingly, proceeding as follows:
 
 10. Load the plugin scripts.					*load-plugins*
 	This does the same as the command: >
-		:runtime! plugin/**/*.vim
-		:runtime! plugin/**/*.lua
+		:runtime! plugin/**/*.\(vim\|lua\)
 <	The result is that all directories in 'runtimepath' will be searched
 	for the "plugin" sub-directory and all files ending in ".vim" or
 	".lua" will be sourced (in alphabetical order per directory),
-	also in subdirectories. First "*.vim" are sourced, then "*.lua" files.
+	also in subdirectories. First "*.vim" are sourced, then "*.lua" files,
+	per directory.
```

Comment:
As written, this kinda contradicts the sentence in `repeat.txt` -- here it sounds like "all vim files, then all lua files"?

I'd recommend writing one (longer) sentence and just copy-pasting it in both places, to make sure they're consistent.

Reply:
Well currently this is what this PR implements, but yes, this needs to be a looser description to match what we have for `:runtime`.

Reply:
To be clear, I'm fine with either behavior (until presented with a compelling concrete and actual example where one would be breaking); I'd just prefer _stricter_  and _matching_ description of the behavior implemented here.